README.md

```diff
@@ -178,10 +178,13 @@ The credentials files use YAML syntax, and the usage will vary depending on the
     #   username: admin
     #   password: password
 
-Custom report
--------------
+HTML report
+-----------
 
-By default a custom HTML report will be written to results/index.html. If you wish this to be located elsewhere, or have a different filename, you can specify the --webqareport command line option.
+If the [pytest-html](https://pypi.python.org/pypi/pytest-html) plugin is installed
+then the HTML reports will include additional information such as the failing URL,
+screenshot, and page source. For Sauce Labs, a link to the job and inline video
+will also be included.
```

Comment:
It might be worth adding that you need to specify the `--html` option in order to generate a report. I know that's documented for the pytest-html plugin, but with pytest-mozwebqa that used to be included by default, without needing an additional argument. That caught me while testing, and I had to look at the docs for pytest-html to figure out why I wasn't getting a report.

Do you think it's worth adding that feature back in, so that if the pytest-html plugin is installed we generate a report by default?